### PR TITLE
docs(snapshot.md): fixed broken urls again using absolute path

### PR DESCRIPTION
**Pull request by vikgaur**

### CODE REVIEW

File: reference/snapshot.md [hunk 1]
```diff
@@ -29,7 +29,7 @@ Currently, Mayastor supports the following operations related to volume snapshot
 ## Prerequisites
 
 
-1. Deploy and configure Mayastor by following the steps given [here](quickstart/deploy-mayastor.md) and create disk pools. 
+1. Deploy and configure Mayastor by following the steps given [here](https://mayastor.gitbook.io/introduction/quickstart/deploy-mayastor.md) and create disk pools.
```
Comment (tiagolobocastro) [inline reference/snapshot.md:32]: Previous PR had used relative path, example: `../` 
If that works it'd be better IMHO

**Comment by avishnu** [comment]
There is a conflict in your branch that needs to be resolved. Most likely, you haven't rebased your local branch with the remote.

**Comment by vikgaur** [comment]
@avishnu @tiagolobocastro I've fixed the conflict. Please review.

File: reference/snapshot.md (hunk 1 above)
Comment (vikgaur) [reply reference/snapshot.md:32]: I've tried using the relative path but it's not working. So, I've used the absolute path to fix the issue.

Comment (avishnu) [reply reference/snapshot.md:32]: The relative path change is already live, but not working. Not sure of the reason. Also noticed that elsewhere in the docs, we seem to have used absolute paths (except SUMMARY which is at top-level).

**Review by avishnu — APPROVED** [review]
lgtm

**Review by tiagolobocastro — APPROVED** [review]

**Pull request merged**